MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMovies-ViewController.swift

```diff
@@ -6,56 +6,62 @@
 //
 
 import UIKit
-import SDWebImage
 
 class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
-    
-    
+
     private let searchMoviesViewModel = SearchMoviesViewModel()
-    
-    
-    @IBOutlet weak var searchMovieTableView: UITableView!
-    
+
+    @IBOutlet private weak var searchMovieTableView: UITableView!
     override func viewDidLoad() {
         super.viewDidLoad()
-        let nib = UINib(nibName: "SearchMovieTableViewCell", bundle: nil)
-        searchMovieTableView.register(nib, forCellReuseIdentifier: "reuseCell")
+        let nib = UINib(nibName: Constants.Identifiers.nibIdentifier, bundle: nil)
+        searchMovieTableView.register(nib, forCellReuseIdentifier: Constants.Identifiers.tableCellReuseIdentifier)
         searchMovieTableView.dataSource = self
         searchMovieTableView.delegate = self
         searchMoviesViewModel.fetchSearchedMovies(completion: { [weak self] result in
```

Comment:
can change this to a trailing closure😄
searchMoviesViewModel.fetchSearchedMovies { [weak self] result in


 
            // Handle result
            DispatchQueue.main.async {
                self?.searchMovieTableView.reloadData()
            }
        }